pib/remote.py

```diff
@@ -0,0 +1,124 @@
+import boto3
+import botocore.exceptions
+import json
+import os
+import shlex
+from pathlib import Path
+from subprocess import Popen, PIPE, CalledProcessError
+
+
+def get_aws_acct_id():
+    return boto3.client('sts').get_caller_identity().get('Account')
+
+
+def get_env():
+    branch = os.getenv('TRAVIS_BRANCH', '').lower()
+    if branch.startswith('env/'):
+        return branch.split('/')[1]
+    else:
+        raise ValueError('Not an environment branch. Expected format env/$name but found {0}'.format(branch))
+
+
+def merge_dicts(defaults, overrides):
+    res = {}
+    res.update(defaults)
+    res.update(overrides)
+    return res
+
+
+def run(cmd, allowed_exit_codes={0}, **kwargs):
+    if type(cmd) is str:
+        cmd = shlex.split(cmd)
+
+    merged = merge_dicts({'stdout': PIPE, 'bufsize': 1, 'universal_newlines': True}, {})
```

Comment:
This seems either superfluous or wrong (perhaps last argument supposed to be `kwargs`)?